### ISSUE TITLE
Switch to static site generator?

### ISSUE THREAD

**Issue by aaronsturm** [typeorm/typeorm.github.io#30]
I've often tried to use Google to search for terms in the documentation. The issue though is that Google is unable to index the site well with its current setup.

![image](https://user-images.githubusercontent.com/4074576/63440235-522ab080-c3e4-11e9-9680-ed39bd1e4cb4.png)

Would there be any interest in moving the site over to a solution like [GastbyJS](https://www.gatsbyjs.org/) to increase SEO? I'd be willing to help with that if interested.

**Comment by MarZab** [comment]
I agree, finding documentation is impossible, there is not even a search function on the page itself.

**Comment by OrkhanAlikhanov** [comment]
@MarZab I posted a quick solution, might be helpful https://github.com/typeorm/typeorm.github.io/issues/16#issuecomment-554615082

**Comment by MarZab** [comment]
Yeah @OrkhanAlikhanov I've been using your gitbook pages for a while now and Google seems to like them better as well - showing them ahead of official pages for most queries - that is probably not ideal for the project long term even if the pages are kept updated.

cc @pleerock 
https://github.com/typeorm/typeorm.github.io/issues/16#issuecomment-554615082
https://github.com/typeorm/typeorm/pull/6180

**Comment by tylerlwsmith** [comment]
I just [submitted a PR](https://github.com/typeorm/typeorm.github.io/pull/36) to the documentation that would partially solve this issue. Google doesn't index hash URLs in single page applications, so the pull request reconfigures Vue Router to not use hashes.

Since the documentation is still a single page app, Google still wouldn't index as quickly as a static site, but this change would allow Google to begin to index the documentation without having to rebuild the whole site.
